sagemaker_model_monitor/model_monitor_batch_transform/SageMaker-ModelMonitoring-Batch-Transform-Model-Quality-With-SageMaker-Pipelines-On-schedule.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>Please only use present tense (this could -&gt; this can). Make sure this applies to the rest of the notebook and other notebooks in this PR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3639/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Done

Reply:
<p>Doesn't look like it changed to 'this can'? At least, from my review view</p>

Reply:
DOne